sympy/physics/continuum_mechanics/cable.py

```diff
@@ -0,0 +1,334 @@
+        if len(self._supports) != 0:
+            raise ValueError("Supports already exist")
+
+        elif len(args) != 2:
+            raise ValueError("Pass only two arguments: left support and right support")
+
+        elif args[0][0] == args[1][0]:
+            raise ValueError("Supports can not have the same label")
+
+        elif args[0][1] == args[1][1] and args[0][2] == args[1][2]:
+            raise ValueError("Supports can not be at the same position")
+
+        elif args[0][1] > args[1][1]:
+            raise ValueError("The x coordinate of left support should be less than its right support counterpart")
+
+        else:
+            for i in args:
+                label = i[0]
+                x = sympify(i[1])
+                y = sympify(i[2])
+
+                self._supports[label] = [x, y]
+
+                if len(self._left_support) == 0:
+                    self._left_support.append(x)
+                    self._left_support.append(y)
+
+                else:
+                    self._right_support.append(x)
+                    self._right_support.append(y)
+
+                self._support_labels.append(label)
+
+    def remove_supports(self):
+        """
+        This method removes the left and right supports, along with
+        any existing load(s).
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.cable import Cable
+        >>> c = Cable()
+        >>> c.add_supports(('A', 3, 4), ('B', 4, 5))
+        >>> c.supports
+        {'A': [3, 4], 'B': [4, 5]}
+        >>> c.remove_supports()
+        >>> c.supports
+        {}
+        """
+        if len(self._supports) == 0:
+            raise ValueError("There are no supports to remove")
+
+        else:
+            self._supports.clear()
+            self._left_support.clear()
+            self._right_support.clear()
+            self._loads.clear()
+            self._loads_position.clear()
+            self._length = 0
+
+    def change_support(self, label, new_support):
+        """
+        This method changes the mentioned support with a new support.
+
+        Parameters
+        ==========
+        label: String or symbol
+            The label of the support to be changed
+
+        new_support: Tuple of the form (new_label, x, y)
+            new_label: String or symbol
+                The label of the new support
+
+            x: Sympifyable
+                The x-coordinate of the position of the new support.
+
+            y: Sympifyable
+                The y-coordinate of the position of the new support.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.cable import Cable
+        >>> c = Cable()
+        >>> c.add_supports(('A', 3, 4), ('B', 4, 5))
+        >>> c.supports
+        {'A': [3, 4], 'B': [4, 5]}
+        >>> c.change_support('B',('C',5,6))
+        >>> c.supports
+        {'A': [3, 4], 'C': [5, 6]}
+        """
+        if label not in self._supports:
```

Comment:
As you mentioned in the meeting, throw another error when an existing load goes out of range on changing a support.